### PR TITLE
Tarte citron meringuée

### PR DESCRIPTION
**Pull request by gbouv**

### CODE REVIEW

File: recettes/sucrees/tarte_citron_meringuee.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Tarte au citron meringuée
+
+---
+
+### Ingrédients
+
+- [ ] 1 (pâte sablée)[./pate_sablee.md]
```
Comment (encoreunefeuille) [inline recettes/sucrees/tarte_citron_meringuee.md]: I do not think this is the right way to add a link, might be wrong though

File: recettes/sucrees/tarte_citron_meringuee.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+---
+
+### Recette
+
+- Préparer la pâte sablée en utilisant 2 jaunes plutôt qu'un oeuf entier. Réserver les 2 blancs restant pour la meringue
+- Faire pré-cuire la pâte 10min à 180°C
+
+###### Crème au citron
+
+- Mélanger 2 oeufs entiers et 2 jaunes (réserver les 2 blancs restant)
```
Comment (encoreunefeuille) [inline recettes/sucrees/tarte_citron_meringuee.md:30]: restants au pluriel

File: recettes/sucrees/tarte_citron_meringuee.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+- Ajouter la crème fraiche et le sucre
+- Ajouter le zeste des citrons ainsi que leur jus
+- Bien battre pour homogénéiser le tout et verser le mélange sur la pâte pré-cuite
+- Cuisson 30min à 180°C
+
+###### Meringue
+
+- Battre les 4 blancs d'oeufs restants neige
```
Comment (encoreunefeuille) [inline recettes/sucrees/tarte_citron_meringuee.md]: "en" neige

File: recettes/sucrees/tarte_citron_meringuee.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+- Ajouter le sucre glace petit-à-petit en continuant de battre
```
Comment (encoreunefeuille) [inline recettes/sucrees/tarte_citron_meringuee.md]: pas sûre qu'il y a des tirets dans petit à petit, can you verify?

File: recettes/sucrees/tarte_citron_meringuee.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+- Déposer la meringue sur la tarte à l'aide d'une poche à douille
+- Cuisson 10min à 180°C. Il est possible d'ajouter 2-3 minutes en mode grill pour faire brunir la meringue 
+- Laisser la tarte dans le four jusqu'à 40 minutes pour avoir une meringue bien croquante
```
Comment (encoreunefeuille) [inline recettes/sucrees/tarte_citron_meringuee.md]: 40min

File: recettes/sucrees/tarte_citron_meringuee.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+###### Crème au citron
+
+- [ ] 180g sucre
+- [ ] 2 citrons
+- [ ] 4 oeufs
+- [ ] 20cl crème liquide
+
+###### Meringue
+
+- [ ] 2 oeufs
```
Comment (encoreunefeuille) [inline recettes/sucrees/tarte_citron_meringuee.md]: plus clair je pense de dire ici : 4 blancs d'oeufs

File: recettes/sucrees/tarte_citron_meringuee.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+### Ingrédients
+
+- [ ] 1 (pâte sablée)[./pate_sablee.md]
+
+###### Crème au citron
+
+- [ ] 180g sucre
+- [ ] 2 citrons
+- [ ] 4 oeufs
```
Comment (encoreunefeuille) [inline recettes/sucrees/tarte_citron_meringuee.md]: 2 oeufs et 2 jaunes d'oeufs

**Review by encoreunefeuille — CHANGES_REQUESTED** [review]

File: recettes/sucrees/tarte_citron_meringuee.md (hunk 4 above)
Comment (gbouv) [reply recettes/sucrees/tarte_citron_meringuee.md]: Yup that's right bro

File: recettes/sucrees/tarte_citron_meringuee.md (hunk 6 above)
Comment (gbouv) [reply recettes/sucrees/tarte_citron_meringuee.md]: Yeah but that makes things unclear above when we say 2 oeufs for the crème au citron. I think you're right this is better, but I'll change the above as well

File: recettes/sucrees/tarte_citron_meringuee.md (hunk 7 above)
Comment (gbouv) [reply recettes/sucrees/tarte_citron_meringuee.md]: Ah; yeah; that makes sense then

File: recettes/sucrees/tarte_citron_meringuee.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+- Ajouter le zeste des citrons ainsi que leur jus
+- Bien battre pour homogénéiser le tout et verser le mélange sur la pâte pré-cuite
+- Cuisson 30min à 180°C
+
+###### Meringue
+
+- Battre les 4 blancs d'oeufs restants en neige
+- Ajouter le sucre glace petit à petit en continuant de battre
+- Déposer la meringue sur la tarte à l'aide d'une poche à douille
+- Cuisson 10min à 180°C. Il est possible d'ajouter 2-3 minutes en mode grill pour faire brunir la meringue
```
Comment (encoreunefeuille) [inline recettes/sucrees/tarte_citron_meringuee.md]: 2-3min

**Review by encoreunefeuille — APPROVED** [review]
just a nit otherwise lgtm 👍

**Pull request merged**